src/packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -1,11 +1,10 @@
-import path from 'path'
-import { Introspect } from '../Introspect'
+import { DbPull } from '../commands/DbPull'
 import { consoleContext, Context } from './__helpers__/context'
-import {
-  SetupParams,
-  setupPostgres,
-  tearDownPostgres,
-} from '../utils/setupPostgres'
+// import
```

Comment:
Is this intended

Reply:
It's not used, doesn't change anything, seems only the code using it was commented (a long time ago?)